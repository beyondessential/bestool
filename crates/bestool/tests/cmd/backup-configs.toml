bin.name = "bestool"
args = "--log-timeless tamanu --root ./tamanu backup-configs --write-to ./"
