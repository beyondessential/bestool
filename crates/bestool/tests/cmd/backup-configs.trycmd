```console
rm -f 1970-01-01_0000-localhost.config.zip
bestool --log-timeless tamanu --root ./tamanu backup-configs --write-to ./ --test-skip-caddy
```
