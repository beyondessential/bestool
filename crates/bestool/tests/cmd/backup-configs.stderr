 WARN bestool::actions::tamanu::backup_configs: Skipping production.json5 while archiving: the file is not found
